core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java

```diff
@@ -834,7 +834,6 @@ public synchronized void freeSpace(long sessionId, long minContiguousBytes,
           spaceFreed += blockMeta.getBlockSize();
         } catch (BlockDoesNotExistException e) {
           LOG.warn("Failed to evict blockId {}, it could be already deleted", blockToDelete);
-          continue;
```

Comment:
why this one is not needed? continue to evict other blocks which may exist looks correct?

Reply:
This continue is for the outer while(true) loop which should stop when eviction finished or no any block that is available to evict

Reply:
+1, I'm also thinking this continue is necessary to evict other blocks